tools/releaseBuild/yaml/releaseBuild.yml

```diff
@@ -70,6 +70,8 @@ stages:
             LINUX_BUILDNAME: 'ubuntu.16.04-arm64'
           AlpineArm64:
             LINUX_BUILDNAME: 'alpine-arm64'
+          AlpineArm64:
```

Comment:
This doesn't look right. `AlpineArm64` is specified already above. Probably needs to be `AlpineArm`?